models/VolunteerFields.js

```diff
@@ -1,47 +1,24 @@
 // these fields should match the model and formData
 // if it doesn't match then add a mapping array/object set
+
+const common = [
+  "mode",
+  "name",
+  "email",
+  "mobile",
+  "co_mobile",
+  "address",
+  "region",
+  "pin",
+  "notes",
+  "services",
+  "act",
```

Comment:
what is this field for?

Reply:
act for "volunteer" or "kind" , page distinction